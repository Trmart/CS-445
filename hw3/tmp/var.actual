.   .   .   Child:   Id: gy [line: 10]
.   .   .   Child:   Id: gy [line: 9]
.   .   .   Child:   Id: py [line: 17]
.   .   .   Child:   Id: py [line: 18]
.   .   .   Child:   Id: sy [line: 21]
.   .   .   Child:   Id: sy [line: 22]
.   .   .   Child:   Id: y [line: 13]
.   .   .   Child:   Id: y [line: 14]
.   Child:   Parm: px of type int [line: 3]
.   .   Child:   Var: x of type int [line: 5]
ERROR(LINKER): A function named 'main()' must be defined.
.   hild:   Compound [line: 4]
.   .   .   hild:   Const 9 [line: 13]
.   .   .   hild:   Const 9 [line: 17]
.   .   .   hild:   Const 9 [line: 21]
.   .   .   hild:   Const 9 [line: 9]
.   .   hild:   Id: gx [line: 8]
Number of errors: 1
Number of warnings: 2
.   .   Sibling: 10  Op: [ [line: 21]
.   .   Sibling: 11  Op: sizeof [line: 22]
.   .   Sibling: 1  Op: [ [line: 9]
.   Sibling: 1  Parm: py of array of type int [line: 3]
Sibling: 1  Var: gy of array of type int [line: 2]
.   .   Sibling: 1  Var: y of array of type int [line: 5]
Sibling: 2  Func: ( returns type void [line: 3]
.   .   Sibling: 2  Op: sizeof [line: 10]
.   .   Sibling: 2  Var: sx of static type int [line: 6]
.   .   Sibling: 3  Id: x [line: 12]
.   .   Sibling: 3  Var: sy of static array of type int [line: 6]
.   .   Sibling: 4  Op: [ [line: 13]
.   .   Sibling: 5  Op: sizeof [line: 14]
.   .   Sibling: 6  Id: px [line: 16]
.   .   Sibling: 7  Op: [ [line: 17]
.   .   Sibling: 8  Op: sizeof [line: 18]
.   .   Sibling: 9  Id: sx [line: 20]
Var: gx of type int [line: 2]
WARNING(12): Variable 'x' may be uninitialized when used here.
WARNING(13): Variable 'y' may be uninitialized when used here.
