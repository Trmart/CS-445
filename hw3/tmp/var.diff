1,5c1,42
< ERROR(LINKER): A function named 'main()' must be defined.
< Number of errors: 1
< Number of warnings: 2
< WARNING(12): Variable 'x' may be uninitialized when used here.
< WARNING(13): Variable 'y' may be uninitialized when used here.
---
> .   .   .   Child: 0  Id: gy [line: 10]
> .   .   .   Child: 0  Id: gy [line: 9]
> .   .   .   Child: 0  Id: py [line: 17]
> .   .   .   Child: 0  Id: py [line: 18]
> .   .   .   Child: 0  Id: sy [line: 21]
> .   .   .   Child: 0  Id: sy [line: 22]
> .   .   .   Child: 0  Id: y [line: 13]
> .   .   .   Child: 0  Id: y [line: 14]
> .   Child: 0  Parm: px of type int [line: 3]
> .   .   Child: 0  Var: x of type int [line: 5]
> .   Child: 1  Compound [line: 4]
> .   .   .   Child: 1  Const 9 of type int [line: 13]
> .   .   .   Child: 1  Const 9 of type int [line: 17]
> .   .   .   Child: 1  Const 9 of type int [line: 21]
> .   .   .   Child: 1  Const 9 of type int [line: 9]
> .   .   Child: 1  Id: gx [line: 8]
> Number of errors: 0
> Number of warnings: 6
> .   .   Sibling: 10  Op: [ [line: 21]
> .   .   Sibling: 11  Op: sizeof [line: 22]
> .   .   Sibling: 1  Op: [ [line: 9]
> .   Sibling: 1  Parm: py of array of type int [line: 3]
> Sibling: 1  Var: gy of array of type int [line: 2]
> .   .   Sibling: 1  Var: y of array of type int [line: 5]
> Sibling: 2  Func: main returns type void [line: 3]
> .   .   Sibling: 2  Op: sizeof [line: 10]
> .   .   Sibling: 2  Var: sx of static type int [line: 6]
> .   .   Sibling: 3  Id: x [line: 12]
> .   .   Sibling: 3  Var: sy of static array of type int [line: 6]
> .   .   Sibling: 4  Op: [ [line: 13]
> .   .   Sibling: 5  Op: sizeof [line: 14]
> .   .   Sibling: 6  Id: px [line: 16]
> .   .   Sibling: 7  Op: [ [line: 17]
> .   .   Sibling: 8  Op: sizeof [line: 18]
> .   .   Sibling: 9  Id: sx [line: 20]
> Var: gx of type int [line: 2]
> WARNING(2): The variable 'gx' seems not to be used.
> WARNING(2): The variable 'gy' seems not to be used.
> WARNING(5): The variable 'x' seems not to be used.
> WARNING(5): The variable 'y' seems not to be used.
> WARNING(6): The variable 'sx' seems not to be used.
> WARNING(6): The variable 'sy' seems not to be used.
