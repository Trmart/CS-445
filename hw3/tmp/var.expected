ERROR(LINKER): A function named 'main()' must be defined.
Number of errors: 1
Number of warnings: 2
WARNING(12): Variable 'x' may be uninitialized when used here.
WARNING(13): Variable 'y' may be uninitialized when used here.
